kong/tools/queue_schema.lua

```diff
@@ -49,5 +49,12 @@ return Schema.define {
       between = { 0.001, 1000000 }, -- effectively unlimited maximum
       description = "Maximum time in seconds between retries, caps exponential backoff."
     } },
+    { concurrency_limit = {
+      type = "integer",
+      default = 1,
+      one_of = { -1, 1 },
```

Comment:
So, we either create unlimited timers or use just 1 timer?

They may ask why not allow creating 10 timers? A reasonable tradeoff between resources and concurrency.

Reply:
Allowing concurrency_limit to be set to > 1 would be the next phase thing.